embedchain/llm/octoai.py

```diff
@@ -0,0 +1,59 @@
+import os
+from typing import Optional
+
+from langchain.llms.octoai_endpoint import OctoAIEndpoint
+
+from embedchain.config import BaseLlmConfig
+from embedchain.helper.json_serializable import register_deserializable
+from embedchain.llm.base import BaseLlm
+
+@register_deserializable
+class OctoaiLlm(BaseLlm):
```

Comment:
Minor suggestion - `OctoAILlm` name looks more semantically aligned with our other LLMs, please let me know what you think